js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,20 @@
+function abbreviateString(str) {
+  if (typeof (str) !== 'string') {
+    throw Error(`Expected string, got ${typeof str}`);
+  }
+
+  if (str.trim().length < 1) {
+    throw Error('Got empty string.');
+  }
+
+  if (str.split(' ').length === 1) {
+    return str[0].toUpperCase() + str.substring(1);
```

Comment:
why would you do this?
why not simply return the string?
```suggestion
    return str;
```